src/schema/me/bidder_position_mutation.js

```diff
@@ -1,10 +1,82 @@
 // @ts-check
 
-import { GraphQLString, GraphQLFloat, GraphQLNonNull } from "graphql"
+import { GraphQLString, GraphQLFloat, GraphQLObjectType, GraphQLNonNull } from "graphql"
 import { mutationWithClientMutationId } from "graphql-relay"
 
+import config from "config"
 import BidderPosition from "schema/bidder_position"
 
+const { PREDICTION_ENDPOINT } = config
+
+// the description_md must be a function to delay interpolation of string literal
+const biddingErrors = [
+  {
+    id: "ERROR_BID_LOW",
+    gravity_key: "Please enter a bid higher than",
+    header: "Your bid wasn't high enough",
+    description: `Another bidder placed a higher max bid or the same max bid before you did.\
+ Bid again to take the lead.`,
+    description_md: () => `Another bidder placed a higher max bid or the same max bid before you did.  \
+ Bid again to take the lead.`,
+  },
+  {
+    id: "ERROR_SALE_CLOSED",
+    gravity_key: "Sale Closed to Bids",
+    header: "Lot closed",
+    description: "Sorry, your bid wasn’t received before the lot closed.",
+  },
+  {
+    id: "ERROR_LIVE_BIDDING_STARTED",
+    gravity_key: "Live Bidding has Started",
+    header: "Live bidding has started",
+    description: `Sorry, your bid wasn’t received before live bidding started.\
```

Comment:
Do we need both `description` and `description_md`?

Reply:
Not for the current screens. Thought it would be useful in future for displaying plain messages but that might never happen. 🤷‍♂️

Reply:
Maybe we can remove for now then? Just one extra place to care about typos etc.